### DISCUSSION TITLE
Index data compression #9025

### DISCUSSION THREAD

**Discussion by kavindasr** [orientechnologies/orientdb#9947] [General]
I am excited to contribute to issue number #9025, which concerns index compressing in Orientdb.
After researching the issue, I would like to implement a compression algorithm for index data. This would allow for a reduction in the amount of disk space required for index storage, as well as potentially improving query performance by reducing the amount of I/O required to access index data.
I believe that this change would greatly benefit the performance and efficiency of Orientdb, and I am eager to contribute to this effort. If anyone has any feedback or suggestions, please let me know.

**Comment by andrii0lomakin** [comment]
Hi @kavindasr 
That would be interesting.
What kind of help do you need?
